src/gui/optionsdialog.ui

```diff
@@ -1094,6 +1094,13 @@
               </property>
              </widget>
             </item>
+            <item>
+             <widget class="QCheckBox" name="checkUnwantedFolder">
+              <property name="text">
+               <string>Keep unselected files in ".unwanted" folder</string>
```

Comment:
```suggestion
               <string>Keep deselected files in ".unwanted" folder</string>
```
unselected is for "never selected" and in this case it's important because they go in .parts file.

Reply:
> unselected is for "never selected"

Hmm... I didn't know about it. I thought they are the same.

>in this case it's important because they go in .parts file

You're not quite right. There may be a case when the file is on the disk, but it is not selected when adding a torrent.

Reply:
If it's still a good idea to rename it, I'll do it. My English is too bad to judge this.

Reply:
>You're not quite right. There may be a case when the file is on the disk, but it is not selected when adding a torrent.

Files smaller than a piece... right, forgot about them. Better keep the current name then.

Reply:
> Files smaller than a piece... right, forgot about them.

Any existing file.